### PR TITLE
fix: imports re-assessed, spelling fixes, .env.example updated

### PR DESCRIPTION
**Pull request by PowVT**
** All upgradeable contracts have some sort of modification to the initialization function. 
* The process taken was to look at all the imported contracts and determine which ones had `_init` functions attached, by viewing the corresponding OpenZeppelin Github code. Imports that where not being initialized, where there is a corresponding `_init` function on Github were added. Even if the function was empty, following the pattern all the way through ensures there are no gaps in initialization. Where fit the `unchained` versions were used per this [OpenZeppelin resource](https://docs.openzeppelin.com/contracts/4.x/upgradeable#multiple-inheritance)
- LoanCore
   1. Added: `__Pausable_init_unchained()`, `__AccessControlEnumerable_init_unchained()`, and `__UUPSUpgradeable_init_unchained()`  
- OriginationController
   1. Added: `__ReentrancyGuard_init_unchained()`, `__Context_init_unchained()`, `__AccessControlEnumerable_init_unchained()`, and `__UUPSUpgradeable_init_unchained()`
   2. Removed: `import "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";` since we are only using the interface.
- VaultFactory
1. added: `unchained` to __ERC721_init_unchained("Asset Vault", "AV")`, `__ERC721Enumerable_init_unchained()`
2. removed: `import "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";` since it is being inherited by `ERC721EnumerableUpgradeable`.
- ERC721PermitUpgradeable
1. added: `unchained` to `__EIP712_init_unchained(name, "1")`

** Remove 'V2 updates' blurb at the top of RepaymentController.sol, this information is redundant and goes against the natspec pattern

** Various spelling fixes

** update the .env.example to match hardhat.config.ts file

### CODE REVIEW

File: contracts/ERC721PermitUpgradeable.sol [hunk 1]
```diff
@@ -60,7 +60,7 @@ abstract contract ERC721PermitUpgradeable is
      * @param name                  The name of the signing domain.
      */
     function __ERC721PermitUpgradeable_init(string memory name) internal {
-        __EIP712_init(name, "1");
+        __EIP712_init_unchained(name, "1");
         __AccessControlEnumerable_init();
         __UUPSUpgradeable_init();
```
Comment (PowVT) [inline contracts/ERC721PermitUpgradeable.sol:66]: It should be noted I did not add the `_init` statement for the `ERC721Upgradeable` not sure if this is necessary, but it is the one import that did not follow the pattern I took when looking at all the other `_init` statements.

**Comment by PowVT** [comment]
> Small grammar finds:
> 
> * word “on” missing before [“an active loan”](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L88)
> * “passed” should be “past” [here](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L89)
> * confusing [sentence](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L100). maybe replace with “get LoanData to send the current installment parameter value to LoanCore”

Done!

**Comment by Mouzayan** [comment]
Small finds continued:

- should be "loan" not [load](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L143)
- use "is" or "equal" instead of [comma](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L177)
- "equals" instead of [comma](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L210) after "amount due"
- "toward principal" instead of ["to"](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L216)
- typo at ["without neededing"](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L227)
- typo at "[load data](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L239)" should be "loan
- maybe use "collect" instead of "[gather](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L243)"
and [here](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L218) and [here](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L270).
- NATSPEC comments from line [254](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L254) to line 266 seem to be remnants from sthng else and do not describe view function `amountToCloseLoan(uint256 loanId)`.  This function is view only and does not "[Pay off...](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/RepaymentController.sol#L258)".

**Comment by Mouzayan** [comment]
Finds in `PromissoryNote.sol` :

- [imports](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/PromissoryNote.sol#L17) not used anywhere in this contract 
- "admin role" vs. "[default admin role](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/PromissoryNote.sol#L39)"
- strange "a" character at start of this [line](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/PromissoryNote.sol#L75). indentation also needs to be fixed upon removal.
- should a "tokenId" variable get defined and returned [here](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/PromissoryNote.sol#L119) as described on line 113 (even if it's equal to loanId)? 
- comment from line [123](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/PromissoryNote.sol#L123) to 126 is copy / paste remnant.

File: contracts/LoanCore.sol [hunk 2]
```diff
@@ -109,6 +109,9 @@ contract LoanCore is
         // only those with FEE_CLAIMER_ROLE can update or grant FEE_CLAIMER_ROLE
         __AccessControlEnumerable_init();
         __UUPSUpgradeable_init_unchained();
+        __Pausable_init_unchained();
+        __AccessControlEnumerable_init_unchained();
```
Comment (kkennis) [inline contracts/LoanCore.sol:113]: I don't think we need this, see line 110

File: contracts/LoanCore.sol [hunk 3]
```diff
@@ -281,7 +284,7 @@ contract LoanCore is
     }
 
     /**
-     * @notice Roll over a loan, atomically closing one and re-opening a new one with the
+     * @notice Rollover a loan, atomically closing one and re-opening a new one with the
```
Comment (kkennis) [inline contracts/LoanCore.sol:287]: I think grammatically the separate words makes sense here. "Rollover" = noun, "roll over" = verb

File: contracts/OriginationController.sol [hunk 4]
```diff
@@ -122,6 +121,10 @@ contract OriginationController is
         __EIP712_init("OriginationController", "2");
         __AccessControlEnumerable_init();
         __UUPSUpgradeable_init_unchained();
+        __ReentrancyGuard_init_unchained();
+        __Context_init_unchained();
+        __AccessControlEnumerable_init_unchained();
+        __UUPSUpgradeable_init_unchained();
```
Comment (kkennis) [inline contracts/OriginationController.sol:127]: Again some of these look like repeats of the above? I think the only one we need is `ReentrancyGuard`.

**Comment by Mouzayan** [comment]
`OriginationController.sol` finds:

- "has" vs. "[should have](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L50)". "of creating" vs. "[to create](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L51)"
- indentation of definitions in NATSPEC is inconsistent, lines [112](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L112) to 118.
- indentation inconsistency lines [137](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L137) and 139 
- "a new loan to be created" vs . "[a new loan to create](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L326)"
- word "in" to replace "as" at the end of the [line ](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L326)
- "a counterparty's" vs."a counterparties'" [here](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L432)
- same as above: "a counterparty's" vs."a counterparties'" [here](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L447)
- inconsistent indentation [here](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L463)
- inconsistent indentation [here](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L635). does not align w/ lines 622 and 624.
- no need for the word "between" before  ["either 0" ](https://github.com/Non-fungible-Technologies/v2-contracts/blob/5d3677e92951408600654f68e8f43350e704960a/contracts/OriginationController.sol#L648)

File: .env.example [hunk 5]
```diff
@@ -1,2 +1,5 @@
-INFURA_API_KEY=zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
 MNEMONIC=here is where your twelve words mnemonic should be put my friend
+ADMIN=0x0000000000000000000000000000000000000000
```
Comment (Mouzayan) [inline .env.example:2]: In the latest AccessControl PR, ADMIN address was updated to 
`0xa49807205ce4d355092ef5a8a18f56e8913cf4a201fbe287825b095693c21775`

Comment (kkennis) [reply .env.example]: `0xa49807205ce4d355092ef5a8a18f56e8913cf4a201fbe287825b095693c21775` is not a valid Ethereum address?

File: test/OriginationController.ts [hunk 6]
```diff
@@ -227,7 +227,7 @@ describe("OriginationController", () => {
             const MockOriginationController = await hre.ethers.getContractFactory("MockOriginationController", other);
             await expect(
                 hre.upgrades.upgradeProxy(originationController.address, MockOriginationController),
-            ).to.be.revertedWith("Ownable: caller is not the owner");
+            ).to.be.revertedWith("AccessControl: account 0x88cde74d0d35369cf2253ef8353581f750b49f1f is missing role 0xa49807205ce4d355092ef5a8a18f56e8913cf4a201fbe287825b095693c21775");
```
Comment (kkennis) [inline test/OriginationController.ts:230]: Remember, we shouldn't be hardcoding accounts or roles here....I have these fixed in my PR so I'd just revert `OriginationController.ts` back to its original state

**Pull request merged**